docs/src/components/MarkdownPage/AdditionalLinks.js

```diff
@@ -174,6 +203,20 @@ const AdditionalLinks = (props: Props) => {
           </BpkLink>
         )}
 
+        {/* Android compose GitHub link */}
+        {platform && platform === PLATFORMS.compose && githubPath && (
+          <BpkLink
+            href={`https://github.com/Skyscanner/backpack-android/tree/main/backpack-compose/src/main/java/net/skyscanner/backpack/compose/${githubPath}`}
```

Comment:
Looks like the path is `kotlin` and not `java` in the path

```suggestion
            href={`https://github.com/Skyscanner/backpack-android/tree/main/backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/${githubPath}`}
```